### ISSUE TITLE
gridview margin issues?

### ISSUE THREAD

**Issue by HerbFargus** [jacobfk20/EmulationStation#6]
after I created the following in my theme (carbon):

```
        <container name="md_grid_margin">
            <size>.05 .05</size>
        </container>
```

It basically went black, there was a cursor blinking really fast in the upper left hand corner and then after a short time the pi rebooted. it may just be due to corruption of my SD card though.

anyways despite the issue with the reboot, I think there are some inconsistencies with this theme option.

correct me if I'm wrong but from what I understand the md_grid_margin is supposed to act as a buffer between each boxart vertically and horizontally?

if I change the size to

```
            <size>.07 0</size>
```

then the first row has a large buffer, then it has the proper size buffer underneath the other rows. 

if I set it to this size

```
            <size>0 .07</size>
```

then there is a large gap at the bottom of the grid

just seems to inconsistent unless I am misunderstanding what the function is supposed to do. if I set it to 0 0 it seems to be fine

**Comment by jacobfk20** [comment]
I took a look and was able to reproduce what you are talking about and fixed it to match how the X axis calculates it's margin.

I have uploaded a rpi (arm) version on [my drive](https://drive.google.com/open?id=0ByWG5kDXyr5NM1c4UHdQeWcxMG8) already for you. (no x86)

Also, margin is more like padding now because the grid tiles are forced to a specific amount of rows and columns and size, so if you increase the padding, they get smaller to fit into the area.  

I could not reproduce what you did to get the crash though.  What setting did you have for <gridrowsandcolumns>?  Maybe matching that could reproduce it.
